public/css/shared.css

```diff
@@ -2,10 +2,29 @@ body {
   margin: 0;
 }
 
+html {
+  background-color: #efefef;
+}
+
 main {
-  margin: 8px;
+  margin: 10px;
 }
 
 .error {
   color: red;
 }
+
+p.footer {
```

Comment:
Daarvoor is de `footer` tag, je hoeft geen eigen class te maken

Reply:
yes nadenken is af en toe moeilijk